File: apps/chat/system-server/src/main/java/com/example/chat/system/config/DomainServiceConfig.java
package com.example.chat.system.config;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import com.example.chat.domain.service.FriendshipDomainService;

@Configuration
public class DomainServiceConfig {

    @Bean
    public FriendshipDomainService friendshipDomainService() {
        return new FriendshipDomainService();
    }
}

File: apps/chat/system-server/src/main/kotlin/com/example/chat/system/config/DomainServiceConfig.kt
package com.example.chat.system.config

import com.example.chat.domain.service.FriendshipDomainService
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class DomainServiceConfig {

	@Bean
	fun friendshipDomainService() = FriendshipDomainService()
}
